classes/warrior.lua

```diff
@@ -1,17 +1,31 @@
 local AddonName, SAO = ...
 
 local function registerClass(self)
+    local tasteforblood = GetSpellInfo(60503);

```

Comment:
Please rename to `tasteForBlood` (in camelCase).

Reply:
Fixed